### DISCUSSION TITLE
Version Number of Software

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#78] [General]
Hi Lars,

The "header" to the program as well as "About" both ought to show the version number of the release.

I am looking forward to using the program again, now updated several times since my last use, for the latest sample-set I am working on. At the moment I am still processing the sound files.

**Comment by larspalo** [comment]
@Bumblebee001 Yes, you're right. It's a regression due to the name change to GoOdf. It's fixed with 3a9dffc8fdfb9deb33917715e8f30677551cc4ec but it will not be available until next release if one doesn't build from source. Most prominent changes since before is the possibility of multiple selections on the GUI panel display and the possibility to create selection areas to select contained or intersecting elements. I'll try to create a screencast showing some of the new features soon...
